templates/download/raspberry-pi/index.html

```diff
@@ -11,11 +11,11 @@
 {% block content %}
 <section class="p-strip--suru-bottomed">
   <div class="row u-vertically-center">
-    <div class="col-8">
-      <h1>Install Ubuntu Server on a Raspberry Pi 2, 3 or 4</h1>
-      <p>Running Ubuntu Server on your Raspberry Pi is easy. Just pick the OS image you want, flash it onto a microSD card, load it onto your Pi and away you go.</p>
+    <div class="col-7">
+      <h1>Install Ubuntu on a Raspberry Pi 2, 3 or 4</h1>
+      <p>Running Ubuntu on your Raspberry Pi is easy. Just pick the OS image you want, flash it onto a microSD card, load it onto your Pi and away you go.</p>
```

Comment:
Small copy doc change.

```suggestion
      <p>Running Ubuntu on your Raspberry Pi is easy. Just pick the OS image you want, flash it onto a microSD card, load it onto your Pi and away you go.</p>
      <p>First time installing Ubuntu on Raspberry Pi?</p>
      <a class="p-button--positive" href="/tutorials/how-to-install-ubuntu-on-your-raspberry-pi">Follow our tutorial</a> 
```